### PR TITLE
Added let vs var differences

### LINKED ISSUE #9: let vs var in javascript program
Describe different between let and var keyword in javascript.
Describe different through code

### PR DESCRIPTION
**Pull request by sai-karthik-p**
#9 Added Javascript Program for 'let vs var differences' with comments and explanation

### CODE REVIEW

File: let vs var.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+// Difference 3: Creation of property on global window object
+
+// let: does not create a property on global window object when a let variable is created in global scope
+let dog = "rusty";
+console.log(window.dog); //undefined
+
+// var: creates a property on global window object when a var variable is created in global scope
+let fruit = "Orange";
+console.log(window.fruit); //Orange
```
Comment (snehakoshti) [inline let vs var.js:69]: here datatype should be var.
you have given result according to var data-type

File: let vs var.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+// Difference 4: Javascript Hoisting
+// Source: https://developer.mozilla.org/en-US/docs/Glossary/Hoisting
+
+// let: variable declaration is hoisted, but not set to 'undefined'
+console.log(color); //ReferenceError: color is not defined
+let color = "red"; 
+
+// var: variable declaration is hoisted, and set to 'undefined'
+console.log(tree); //undefined
+let tree = "oak";
```
Comment (snehakoshti) [inline let vs var.js]: again here also datatype should be var

File: let vs var.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+// var: is  scoped to "current execution context" AKA a variable's enclosing function or the global scope
+var varPet = "dog";
+console.log(varPet); // dog // (global scope)
+
+if(true) {
+    var varFruit = "apple";
+    console.log(varFruit); // apple // (enclosing function scope)
+}
+console.log(varFruit); //ReferenceError: varFruit is not defined
```
Comment (snehakoshti) [inline let vs var.js]: output in comment is wrong.
both place we will get "apple" output

**Review by snehakoshti — COMMENTED** [review]
please check comment

**Comment by sai-karthik-p** [comment]
@snehakoshti Thanks for pointing out.

1. I've resolved PR comments
2. And also added some code for `enclosing function scope` in var which was missing earlier

**Review by snehakoshti — APPROVED** [review]
Approved

**Comment by snehakoshti** [comment]
PR merged

**Pull request merged**